lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -2280,7 +2280,7 @@ $(document).ready(function () {
     var section = addAboveLI.parent().parent().parent();
     var sectionId = "sectionid" + (nextid++);
-    section.prev('.sectionHeader').parent().after('<div><h3 class="sectionHeader skip"><span aria-hidden="true" class="collapseIcon fa-caret-down"></span><span class="sectionHeaderText"></span><span class="toggleCollapse">' + msg('simplepage.clickToCollapse') + '</span></h3><div class="section"><div class="column"><div class="editsection"><span class="sectionedit"><h3 class="lb-offscreen">' + msg('simplepage.break-here') + '</h3><button type="button" data-merge-id="' + newitem + '" aria-label="' + msg('simplepage.join-items') + '" title="' + msg('simplepage.join-items') + '" class="section-merge-link"><span aria-hidden="true" class="fa-compress fa-edit-icon sectioneditfont"></span></button></span><span class="sectionedit sectionedit2"><a href="/lessonbuilder-tool/templates/#" title="' + msg('simplepage.columnopen') + '" class="columnopen" style="text-decoration: none;"><span aria-hidden="true" class="fa-cog fa-edit-icon sectioneditfont"></span></a></span></div><span class="sectionedit addbottom"><a href="#" title="Add new item at bottom of this column" class="add-bottom"><span aria-hidden="true" class="fa-plus fa-edit-icon plus-edit-icon"></span></a></span><div border="0" role="list" style="z-index: 1;" class="indent mainList"><div class="breakitem breaksection" role="listitem"><span style="display:none" class="itemid">' + newitem + '</span></div></div></div></div></div>');
+    section.prev('.sectionHeader').parent().after('<div><h3 class="sectionHeader skip"><span aria-hidden="true" class="collapseIcon fa-caret-down"></span><span class="sectionHeaderText"></span><span class="toggleCollapse">' + msg('simplepage.clickToCollapse') + '</span></h3><div class="section"><div class="column"><div class="editsection"><span class="sectionedit"><h3 class="lb-offscreen">' + msg('simplepage.break-here') + '</h3><button type="button" data-merge-id="' + newitem + '" aria-label="' + msg('simplepage.join-items') + '" title="' + msg('simplepage.join-items') + '" class="section-merge-link"><span aria-hidden="true" class="fa-compress fa-edit-icon sectioneditfont"></span></button></span><span class="sectionedit sectionedit2"><a href="#" title="' + msg('simplepage.columnopen') + '" class="columnopen" style="text-decoration: none;" data-bs-toggle="modal" data-bs-target="#column-dialog" aria-controls="column-dialog" aria-expanded="false"><span aria-hidden="true" class="fa-cog fa-edit-icon sectioneditfont"></span></a></span></div><span class="sectionedit addbottom"><a href="#" title="Add new item at bottom of this column" class="add-bottom"><span aria-hidden="true" class="fa-plus fa-edit-icon plus-edit-icon"></span></a></span><div border="0" role="list" style="z-index: 1;" class="indent mainList"><div class="breakitem breaksection" role="listitem"><span style="display:none" class="itemid">' + newitem + '</span></div></div></div></div></div>');
```

Comment:
Hey @hornersa, I suggest refactoring the HTML string construction using ES6 template literals. It enhances readability and makes the code cleaner.

Instead of concatenating with +, we can use template literals. With this, it's easier to spot the HTML structure. Let me know what you think.


Here's how I'd do it.
```suggestion
const joinItemsMsg = msg('simplepage.join-items');
const breakHereMsg = msg('simplepage.break-here');
const columnOpenMsg = msg('simplepage.columnopen');

const newHtml = `
<div>
    <h3 class="sectionHeader skip">
        <span aria-hidden="true" class="collapseIcon fa-caret-down"></span>
        <span class="sectionHeaderText"></span>
        <span class="toggleCollapse">${msg('simplepage.clickToCollapse')}</span>
    </h3>
    <div class="section">
        <div class="column">
            <div class="editsection">
                <span class="sectionedit">
                    <h3 class="lb-offscreen">${breakHereMsg}</h3>
                    <button type="button" data-merge-id="${newitem}" aria-label="${joinItemsMsg}" title="${joinItemsMsg}" class="section-merge-link">
                        <span aria-hidden="true" class="fa-compress fa-edit-icon sectioneditfont"></span>
                    </button>
                </span>
                <span class="sectionedit sectionedit2">
                    <a href="#" title="${columnOpenMsg}" class="columnopen" style="text-decoration: none;" data-bs-toggle="modal" data-bs-target="#column-dialog" aria-controls="column-dialog" aria-expanded="false">
                        <span aria-hidden="true" class="fa-cog fa-edit-icon sectioneditfont"></span>
                    </a>
                </span>
            </div>
            <span class="sectionedit addbottom">
                <a href="#" title="Add new item at bottom of this column" class="add-bottom">
                    <span aria-hidden="true" class="fa-plus fa-edit-icon plus-edit-icon"></span>
                </a>
            </span>
            <div border="0" role="list" style="z-index: 1;" class="indent mainList">
                <div class="breakitem breaksection" role="listitem">
                    <span style="display:none" class="itemid">${newitem}</span>
                </div>
            </div>
        </div>
    </div>
</div>
`;

section.prev('.sectionHeader').parent().after(newHtml);

```